mmv1/third_party/terraform/tests/resource_composer_environment_test.go.erb

```diff
@@ -1676,6 +1710,76 @@ resource "google_compute_subnetwork" "test" {
 `, envName, network, subnetwork)
 }
 
+func testAccComposerEnvironment_composerV2HighResilience(envName, network, subnetwork string) string {
+	return fmt.Sprintf(`
+resource "google_composer_environment" "test" {
+  name   = "%s"
+  region = "us-east1"
+
+	config {
+		node_config {
+			network          = google_compute_network.test.self_link
+			subnetwork       = google_compute_subnetwork.test.self_link
+		}
+
+		software_config {
+			image_version = "composer-2-airflow-2"
+		}
+
+		workloads_config {
+			scheduler {
+				cpu         = 1.25
+				memory_gb   = 2.5
+				storage_gb  = 5.4
+				count       = 2
+			}
+			web_server {
+				cpu         = 1.75
+				memory_gb   = 3.0
+				storage_gb  = 4.4
+			}
+			worker {
+				cpu         = 0.5
+				memory_gb   = 2.0
+				storage_gb  = 3.4
+				min_count   = 2
+				max_count   = 5
+			}
+			<% unless version == "ga" -%>
```

Comment:
Nit: is this relevant to resilience_mode? If not, it might be simplest to just remove this block.

Reply:
Removed. This should make the test cleaner.